docs/_sass/application.scss

```diff
@@ -82,3 +83,21 @@ $govuk-assets-path: "/govuk-google-analytics/assets/";
   color: govuk-colour("dark-grey");
   @include govuk-font(19);
 }
+
+.highlight-json {
+  padding: govuk-spacing(2);
+  border: solid 1px $govuk-border-colour;
+  background: govuk-colour("light-grey");
+}
+
+.highlight-json__brace {
+
+}
+
+.highlight-json__key {
+  color: govuk-colour("dark-blue");
+}
+
+.highlight-json__value {
+  color: govuk-colour("red");
```

Comment:
Just noticed as well, this doesn't meet our accessibility requirements for colour contrast against the grey background used.

Reply:
No worries - yeah I think that colour is fine when on a white background, but the grey made it inaccessible.